lfs/pointer_clean.go

```diff
@@ -18,10 +18,10 @@ type cleanedAsset struct {
 	*Pointer
 }
 
-func PointerClean(reader io.Reader, fileName string, fileSize int64, cb progress.CopyCallback) (*cleanedAsset, error) {
+func PointerClean(reader io.Reader, fileName string, fileSize int64, cb progress.CopyCallback) (*cleanedAsset, int64, error) {
```

Comment:
Does this need to return another `int64`? You should be able to get the actual blob size from `*cleanedAsset`.